### PR TITLE
Feature/loader updates

### PR DESCRIPTION
**Pull request by elizoller**

### CODE REVIEW

File: spec/helpers/handle_helper_spec.rb [hunk 1]
```diff
@@ -1,21 +1,20 @@
 require "spec_helper"
 include HandleHelper
 
-describe HandleHelper, unless: $in_travis do
+describe HandleHelper do
   let(:bills_file) { FactoryGirl.create(:bills_complete_file) }
-  `mysql -u "#{ENV["HANDLE_TEST_USERNAME"]}" < "#{Rails.root}"/spec/fixtures/files/handlesTEST.sql`
```
Comment (dgcliff) [inline spec/helpers/handle_helper_spec.rb]: @elizoller These need to stay.

If you use non-test values in specs, and they were run on production, you'd be adding test values to the production handle database.

By putting in the values in Travis.yml, you're absolving yourself of that risk though.

Comment (elizoller) [inline spec/helpers/handle_helper_spec.rb]: They are still there just moved into the before :all loop, is that ok?

Comment (dgcliff) [inline spec/helpers/handle_helper_spec.rb]: Sorry, should've elaborated. HANDLE_TEST values and HANDLE (non-test) values are separated for a reason, and can't be used interchangeably (for the production reason I outlined above.

Unless I'm reading incorrectly, HANDLE_TEST was replaced by by the non test values in this spec file.

If we include travis, there are 8 handle databases

Dev - handles (fake), handles_test (fake)
Staging - handles (fake), handles_test (fake)
Travis - handles (fake), handles_test (fake)
Prod - handles (real!!), handles_test (fake)

Comment (elizoller) [inline spec/helpers/handle_helper_spec.rb]: Then I'm still faced with the same issue of getting the test values into the jobs unless i change the jobs to accept the client like handle_helper does

File: spec/mailers/loader_mailer_spec.rb [hunk 2]
```diff
@@ -3,14 +3,40 @@
 
   describe "load_alert" do
     before :each do
-      #FactoryGirl.create_list(:marcom_load)
+      `mysql -u "#{ENV["HANDLE_USERNAME"]}" < "#{Rails.root}"/spec/fixtures/files/handlesTEST.sql`
+      @client = Mysql2::Client.new(:host => "#{ENV["HANDLE_HOST"]}", :username => "#{ENV["HANDLE_USERNAME"]}", :password => "#{ENV["HANDLE_PASSWORD"]}", :database => "#{ENV["HANDLE_DATABASE"]}")
```
Comment (dgcliff) [inline spec/mailers/loader_mailer_spec.rb]: I think this is the offender @elizoller

**Pull request merged**